tinyms/data/loader.py

```diff
@@ -31,10 +31,11 @@
 from mindspore.dataset.engine import *
 from mindspore.mindrecord import FileWriter
 from .utils import generate_image_list, load_img
+from tinyms import data as ds
```

Comment:
There is no need to import data again, all packages have been imported by `from mindspore.dataset.engine import *`